libraries/ms-drivers/test/test_drv120_relay.c

```diff
@@ -0,0 +1,38 @@
+// Brief test sequence for DRV120
+
+#include "drv120_relay.h"
+#include "log.h"
+#include "ms_test_helpers.h"
+
+// Pin used to enable/disable relay
+const GpioAddress TEST_DRV120_PIN = { GPIO_PORT_A, 8 };
+
+void setup_test(void) {
+  gpio_init();
+}
+
+void teardown_test(void) {}
+
+void test_drv120_relay(void) {
+  // Variable used later to get GPIO state
+  GpioState state = GPIO_STATE_LOW;
+
+  LOG_DEBUG("Initializing relay \n");
```

Comment:
Minor nit: you don't need the space in front of the \n here and in the other logs.